migration/utils.py

```diff
@@ -11,3 +15,17 @@ def find_entity_by_id(id: int, entities: list[T]) -> T | None:
         if entity.id == id:
             return entity
     return None
+
+
+def convert_csv_to_int_list(csv_string: str) -> list[int]:
```

Comment:
looks to me this is convert_string_to_int_list, instead of convert_csv_to_int_list?

Reply:
I see. Then keep the name since the expectation is always for comma separated strings.